spring-project/src/main/java/com/ivoyant/spring_project/service/FeedbackService.java

```diff
@@ -0,0 +1,27 @@
+package com.ivoyant.spring_project.service;
+
+import com.ivoyant.spring_project.dao.FeedbackDAO;
+import com.ivoyant.spring_project.model.EmployeeFeedback;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class FeedbackService {
+
+    @Autowired
+    private FeedbackDAO feedbackDAO;
+
+    public void addFeedback(EmployeeFeedback feedback) {
+        feedbackDAO.addFeedback(feedback);
```

Comment:
DAO is not required 
You can keep all DAO logic in Service class

Reply:
updated